app/build.gradle

```diff
@@ -26,4 +26,7 @@ dependencies {
     compile 'com.android.support.constraint:constraint-layout:1.0.0-beta4'
     compile 'me.dm7.barcodescanner:zxing:1.9'
     compile 'org.slf4j:slf4j-android:1.7.22'
+    compile 'com.google.code.gson:gson:2.7'
```

Comment:
Do you need to explicitly require `gson`? I believe `converter-gson` below will do this for us.